### PR TITLE
[PHP] Update and extend setup for auto instrumentation

### PR DESCRIPTION
**Pull request by pdelewski**

### CODE REVIEW

File: content/en/docs/instrumentation/php/automatic.md [hunk 1]
```diff
@@ -45,11 +45,23 @@ function ends it.
 
 ## Setup
 
-1. Install the extension via pickle or [php-extension-installer](https://github.com/mlocati/docker-php-extension-installer):
+1. Install the extension via a. [pickle](https://github.com/FriendsOfPHP/pickle) or b. [php-extension-installer](https://github.com/mlocati/docker-php-extension-installer) (docker specific):
```
Comment (svrnm) [inline content/en/docs/instrumentation/php/automatic.md:48]:
```suggestion
1. Install the extension via [pickle](https://github.com/FriendsOfPHP/pickle) or [php-extension-installer](https://github.com/mlocati/docker-php-extension-installer) (docker specific):
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 2]
```diff
@@ -45,11 +45,23 @@ function ends it.
...
+
+    ```console
+    $ php pickle.phar install --source https://github.com/open-telemetry/opentelemetry-php-instrumentation.git#1.0.0beta1
+    ```
+
+   b.
+
+    ```console
+    $ install-php-extensions open-telemetry/opentelemetry-php-instrumentation@main
+    ```
```
Comment (svrnm) [inline content/en/docs/instrumentation/php/automatic.md:64]:
```suggestion
    * **pickle** can be used to install extensions that are available via http://pecl.php.net, however that's not the case
    for opentelemetry-php-instrumentation yet, so the only way for it is to install directly from source code.
    Following command line shows how to do that using specific version of extension (1.0.0beta1 in this case).

    ```console
    $ php pickle.phar install --source https://github.com/open-telemetry/opentelemetry-php-instrumentation.git#1.0.0beta1
    ```
   
   *  **php-extension-installer**
   ```console
    $ install-php-extensions open-telemetry/opentelemetry-php-instrumentation@main
    ```
```

**Review by svrnm — CHANGES_REQUESTED** [review]
the inlined a. and b. breaks the reading flow, I suggested a different formatting by using normal unordered list items. Please give it a look.

**Comment by svrnm** [comment]
thx for fixing the formatting, I think you need to add more tabs/spaces to the code snippets, since they are not in line with their bullet point:

https://deploy-preview-2151--opentelemetry.netlify.app/docs/instrumentation/php/automatic/#setup

**Comment by pdelewski** [comment]
> thx for fixing the formatting, I think you need to add more tabs/spaces to the code snippets, since they are not in line with their bullet point:
> 
> https://deploy-preview-2151--opentelemetry.netlify.app/docs/instrumentation/php/automatic/#setup

ok, thank you

**Review by svrnm — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Comment by cartermp** [comment]
Merging since you're a maintainer :)

**Pull request merged**